### ISSUE TITLE
Deploy error

### ISSUE THREAD

**Issue by bulgakovk** [arunoda/meteor-up-legacy#743]
Hello, I've got error when I'm deploying my meteor app.
Here is code from command line:

> Building Started: /root/bingo
> 
> => Build Error. Check the logs printed above."

But It's clear above. There are not some logs.

Here is my mup.json:

> {
> // Server authentication info
>  "servers": [
>    {
>      "host": "CHANGED",
>      "username": "root",
>      "password": "CHANGED"
>      // or pem file (ssh based authentication)
>      //"pem": "~/.ssh/id_rsa"
>   }
> ],
> 
> // Install MongoDB in the server, does not destroy local MongoDB on future setup
> "setupMongo": true,
> 
>  // WARNING: Node.js is required! Only skip if you already have Node.js installed on server.
>  "setupNode": true,
> 
>  // WARNING: If nodeVersion omitted will setup 0.10.36 by default. Do not use v, only version >number.
>  "nodeVersion": "0.10.36",
> 
>  // Install PhantomJS in the server
>  "setupPhantom": true,
> 
>  // Show a progress bar during the upload of the bundle to the server. 
>  // Might cause an error in some rare cases if set to true, for instance in Shippable CI
>  "enableUploadProgressBar": true,
> 
>  // Application name (No spaces)
>  "appName": "bingo",
> 
>  // Location of app (local directory)
>  "app": "/root/bingo",
> 
>  // Configure environment
>  "env": {
>    "ROOT_URL": "CHANGED"
>  },
> 
>  // Meteor Up checks if the app comes online just after the deployment
>  // before mup checks that, it will wait for no. of seconds configured below
>  "deployCheckWaitTime": 5
> }

Some help?

**Comment by MasterJames** [comment]
First thing to try that could be a problem is changing the node version to 0.10.40 and updating to meteor 1.2.1.
It seems 0.10.36 doesn't work with meteor 1.2

**Comment by bulgakovk** [comment]
> root@:~/bingo/.deploy# meteor --version
> Meteor 1.2.1
> root@:~/bingo/.deploy# node --version
> v0.10.40
> root@:~/bingo/.deploy# mup deploy
> 
> ## Meteor Up: Production Quality Meteor Deployments
> 
> “ Checkout Kadira!
>   It's the best way to monitor performance of your app.
>   Visit: https://kadira.io/mup ”
> 
> Building Started: ../
> 
> => Build Error. Check the logs printed above.

Doesn't help.

**Comment by bulgakovk** [comment]
> /root/.meteor/packages/meteor->tool/.1.1.10.ki0ccv++os.linux.x86_64+web.browser+web.cordova/mt->os.linux.x86_64/dev_bundle/lib/node_modules/fibers/future.js:245
>                                               throw(ex);
>                                                      ^
> Error: Unable to allocate ArrayBuffer.

Idk why but now I can see logs. It seems I've got not enough memory on VPS. I think, issue can be closed.

**Comment by bulgakovk** [comment]
Yes, it works fine after updating my server RAM and Node.js/Meteor.js versions. Thanks for help!

**Comment by Goawayjose** [comment]
how did you update your server RAM?

**Comment by bulgakovk** [comment]
Just bought more expensive VPS.
